docs/isa-support.rst

```diff
@@ -103,7 +103,7 @@ opcode  src_reg  offset  imm   description
 0x7f    any      0       0x00  dst >>= src                                             Y        Y     Y      rsh-reg
 0x84    0x0      0       0x00  dst = (u32)-dst                                         Y        Y     Y      neg
 0x85    0x0      0       any   call helper function by static ID                       Y        Y     Y      call_unwind_fail
-0x85    0x1      0       any   call PC += imm                                          no       no    no     call_local
+0x85    0x1      0       any   call PC += imm                                          Y        no    Y      call_local
```

Comment:
ubpf does have support for local function calls. (See https://github.com/iovisor/ubpf/actions/runs/8862338746/job/24335181367#step:13:107)

Reply:
Fixed thanks.